modules.d/50boot-animation/module-setup.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# called by dracut
+check() {
+    return 255
+}
+
+# called by dracut
+depends() {
+    # shellcheck disable=SC2043
+    for module in plymouth-ng; do
+        if dracut_module_included "$module"; then
+            splash_backend="$module"
+            break
+        fi
+    done
+
+    if [ -z "$splash_backend" ]; then
+        if [[ -e $dracutsysrootdir$systemdsystemunitdir/plymouth-start.service ]]; then
```

Comment:
Do you know if this condition is true in at least one of the CI containers (Arch, Debian, Gentoo, Debian, Fedora, openSUSE) ? 

In other words is this line tested by the current CI ? 

Thanks !

Reply:
systemd based distros with plymouth installed should have it.

With `dracutsysrootdir=""` or `dracutsysrootdir="/"`:
```
[zozo@fedora ~]$ export systemdsystemunitdir=/usr/lib/systemd/system
[zozo@fedora ~]$ if [[ -e $dracutsysrootdir$systemdsystemunitdir/plymouth-start.service ]]; then echo file exists ; fi
file exists
```